modules/oe_corporate_countries_address/src/EventSubscriber/AvailableCountriesSubscriber.php

```diff
@@ -52,10 +52,12 @@ public function removeDeprecatedCountries(AvailableCountriesEvent $event) {
     // If no available countries are passed, default it to all available ones.
     if (empty($available_countries)) {
       $available_countries = array_column($this->corporateCountryRepository->getCountries(), 'alpha-2');
+      $available_countries = array_combine($available_countries, $available_countries);
```

Comment:
This line ensures the country array is keyed correctly (and is covered in the test). From this:
```
      0 => 'XK',
      1 => 'AX',
      2 => 'BE',
      3 => 'TF',
      4 => 'IT',
```

To this:
```
'XK' => 'XK',
'AX' => 'AX',
'BE' => 'BE',
'TF' => 'TF',
'IT' => 'IT',
```

Reply:
Understood.